lib/InternalRequest.php

```diff
@@ -7,28 +7,52 @@
 class InternalRequest {
     use Struct;
 
+    /** @var Client */
     public $client;
```

Comment:
use `public Client $client` instead of a phpdoc (where appropriate in this class)